sdk/src/main/java/io/opentelemetry/sdk/trace/Samplers.java

```diff
@@ -51,6 +51,20 @@
   // No instance of this class.
   private Samplers() {}
 
+  /**
+   * Returns a {@link Decision} with empty attributes and {@link Decision#isSampled()} returning the
+   * value of the parameter {@code isSampled}.
+   *
+   * <p>This is meant for use by custom {@link Sampler} implementations.
+   *
+   * @param isSampled The value to return from {@link Decision#isSampled()}.
+   * @return A {@link Decision} with empty attributes and {@link Decision#isSampled()} returning the
+   *     value of the parameter {@code isSampled}.
+   */
+  public static final Decision emptyDecision(boolean isSampled) {
+    return isSampled ? ALWAYS_ON_DECISION : ALWAYS_OFF_DECISION;
```

Comment:
I would expect custom `Sampler`s to use something like:
```
Samplers.alwaysOn/Off().shouldSample(...)
```
so attributes can be set

Reply:
I don't understand. Which attributes could be set?

Reply:
That depends on the implementation. For example, jaeger adds their rate limiter to the attributes

Reply:
Of course, for samplers that needs to add attribute, this function cannot be used.
But at least for the negative decision, attributes will AFAIK be ignored, so at least for this case it could be used by any sampler.

Probably a more full-featured version of this PR could also add `singleAttributeDecision(boolean, key, value)` and `attributedDecision(boolean, Map)`. But for the sampler I'm thinking of I would only need `emptyDecision`.

Reply:
About your suggestion with `Samplers.alwaysOn/Off().shouldSample(...)`:

First, it is very verbose and IMHO also not as clear. Second, if you need to do something like `someBoolean ? Samplers.alwaysOn... : Samplers.alwaysOff...` it becomes even twice as verbose.

Reply:
And I still don't understand how attributes could be added using `Samplers.alwaysOn/Off().shouldSample(...)`